### ISSUE TITLE
Can not drag out sidebar when activity bar and sidebar hidden

### ISSUE THREAD

**Issue by jianboy** [microsoft/vscode#92709]
Issue Type: <b>Bug</b>

zend mode is a good function. I can sliding to the right and it show side bar in vscode 1.40.

![4](https://user-images.githubusercontent.com/11532015/76696150-3ca00f80-66c3-11ea-9fff-0b0f6b7ea6ed.gif)

but it can not do it in vscode 1.43
![5](https://user-images.githubusercontent.com/11532015/76696200-afa98600-66c3-11ea-8042-b286b69a7c20.gif)

VS Code version: Code 1.43.0 (78a4c91400152c0f27ba4d363eb56d2835f9903a, 2020-03-09T19:47:57.235Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6500U CPU @ 2.50GHz (4 x 2592)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.89GB (11.89GB free)|
|Process Argv|D:\Program Files\AutoHotkey\Script|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
git-project-manager|fel|1.7.1

</details>

**Comment by boltex** [comment]
I would reword it as "Cannot display the side bar by dragging from the left side of the screen while in zen mode"

**Comment by isidorn** [comment]
Thanks for the suggestion, I have renamed this issue.
I blieve this is an issue with the new layout, thus forwarding to @sbatten

**Comment by joaomoreno** [comment]
@sbatten This is a consequence of our decision here: https://github.com/microsoft/vscode/issues/84315#issuecomment-554126812

**Comment by joaomoreno** [comment]
Should fullscreen veto that?

**Comment by sbatten** [comment]
I think so. The ideal user scenario allows them to do this when it is unambiguous.

**Comment by joaomoreno** [comment]
![you got it](https://media.giphy.com/media/l41lZxzroU33typuU/giphy.gif)

**Comment by jianboy** [comment]
can anybody  fix the bug? It's been six months.

**Comment by jianboy** [comment]
> I'm sorry but the changes in this PR make no sense. It feels you just started deleting code and stopped when your issue got fixed.
> 
> Please follow the discussion in: [#92709 (comment)](https://github.com/microsoft/vscode/issues/92709#issuecomment-607224841) The fix should be related to full screen mode being enabled.

**Hello, @joaomoreno  I hope you can compile the source yourself to experience whether the problem is solved. I debugged it on the master branch and solved it.**

**The bug has nothing to do with full screen**，The problem will also occur when the window is normal size(master source). it can not drag out sidebar **when activity bar and sidebar hidden**. when the activity bar or the sidebar hidden, there is a problem with the **absolute size** of vscode windows 

I try to find the line that caused the bug. There is a problem with the calculation of the two variables in the code, and this judgment is meaningless, so I deleted it and the problem was solved.

```
absoluteSize


	if (this._endSnappingEnabled === endSnappingEnabled) {	
			return;	
		}

```

**Comment by jianboy** [comment]
![7](https://user-images.githubusercontent.com/11532015/94133314-14277800-fe93-11ea-9777-db0ab1905a51.gif)

**Comment by jianboy** [comment]
![8](https://user-images.githubusercontent.com/11532015/94135205-b0eb1500-fe95-11ea-8f3a-2c8a220a02b3.gif)
